Backend/UnitTesting/ITestingUtils.cs

```diff
@@ -12,9 +12,9 @@ interface ITestingUtils
     {
         User createUser();
         Session createSession(User user);
-        Service createService(bool enabled);
-        Claim createClaim(User user, Service service);
+        //Service createService(bool enabled);
+        //Claim createClaim(User user, Service service);
         bool isEqual(string[] arr1, string[] arr2);
-        Client createClientObject();
+        //Client createClientObject();
```

Comment:
What is the reason for these changes?

Reply:
These are creating to project build errors, these classes do not existing in the solution